docs/README.md

```diff
@@ -1,14 +1,30 @@
 # Introduction
 
-Earthly is a build automation tool from the same era as your code. It allows you to execute all your builds in containers. This makes them self-contained, repeatable, portable and parallel. You can use Earthly to create Docker images and artifacts (e.g. binaries, packages, arbitrary files).
+Earthly is a versatile, approachable CI/CD framework that gives you repeatable builds by running each build inside containers *– write builds once, and they will run anywhere*. Earthly has a super simple, instantly recognizable syntax that is easy to write and understand *– like Dockerfile and Makefile had a baby*. And it leverages and augments popular build tools instead of replacing them, *so you don’t have to rewrite all your builds no matter what languages you use*. With Earthly, you can create Docker images and build artifacts (e.g. binaries, packages, and arbitrary files).
```

Comment:
I'm not a fan of the size of this first paragraph; I prefer READMEs with a concise intro.

for example, it feels odd that we repeate `like Dockerfile and Makefile had a baby` in the opening, then again under the `key features/benefits` section.

Reply:
Great feedback. I updated to a shorter intro paragraph as per your recommendation.